src/App.js

```diff
@@ -1,19 +1,160 @@
-import {About, Profile, Home, Login, NotFound} from "./components";
-import {Route, Routes} from "react-router-dom";
-import React from "react";
+import {About, Profile, Home, Login, NotFound, UserDashboard} from "./components";
+import * as React from "react";
+import {
+    Routes,
+    Route,
+    Link,
+    useNavigate,
+    useLocation,
+    Navigate,
+    Outlet,
+} from "react-router-dom";
 
 import './App.css';
 
 function App() {
-  return (
-        <Routes>
-          <Route path="/" element={<Home />} />
-          <Route path ="/Profile" element={<Profile />} />
-          <Route path ="/About" element={<About />} />
-          <Route path ="/Login" element={<Login />} />
-          <Route path="*" element={<NotFound />} />
-        </Routes>
-          );
+    return (
+        <Login />
+    );
 }
 
+//         <Routes>
```

Comment:
why is this all commented out?

Reply:
Only displaying the login page, the other code should be in the multi page branch

Reply:
ok, then should we just remove the commented out code? if the rest of the code is on multi page branch?